content/blog/why-is-redfish®-different-from-other-rest-apis-part-1.md

```diff
@@ -0,0 +1,134 @@
+
+#### What’s inside the protocol specification?
+
+* Boring but necessary topics like the normative framework (definitions, symbols, glossary, typographical conventions, etc.).
+* Things a little more interesting like compliance with other standards (<a href="https://www.odata.org/" target="_blank">Open Data</a>, <a href="https://swagger.io/specification/" target="_blank">Swagger/OpenAPI</a>, etc.).
+* We also find very practical things like the ways to authenticate (basic authentication, session token, etc.) and the endpoints where authentication is necessary (everywhere except at the root `/redfish/v1`).
+* Surprising things like receiving an "error" messages for successful requests. Figure 2 shows the successful erasure of a storage logical volume (LUN) whose response contains the `error{}` object with a message explaining the success of the operation! This type of incongruous response is a consequence of compliance with the <a href="https://www.odata.org/" target="_blank">Open Data</a> standard.
+
+![Figure 2: The error contains a success message !](/img/fig2-successerror.png "Figure 2: The error contains a success message !")
+
+#### Data model
+
+Responses to Redfish® requests consist of JSON packets containing key/value properties defined by the DMTF in a schema file. This name of schema files is contained in the `@odata.type` property that must be present in each response.
+
+For example, the schema defining the root of the Redfish® tree is `#ServiceRoot`. Its full name returned by `curl -sk https://bmc-ip/redfish/v1 | jq '."@odata.type"'` is: `#ServiceRoot.v1_13_0.ServiceRoot`. Appended to the `#ServiceRoot` fragment, a version number (`1_13_0`) and then a subtype which, in this specific case, is identical to the main schema. All schemas are publicly available on the <a href="at https://redfish.dmtf.org/schemas" target="_blank">DMTF website</a> and, are sometimes included in the service itself (see the [Self Sufficient Model](#self-sufficient-model) paragraph below). Note that schema versions can evolve independently of each other.
+
+With a close look to the Redfish® root diagram in (Figure 3), you will notice the presence of endpoints allowing access to the modeling of the server subsystems. For example, the `Chassis{}` object points to `/redfish/v1/Chassis` which contains the exhaustive collection of the URIs (`./{ChassisId}`) modeling the different chassis constituting the server (racks, blades, enclosures, storage enclosures, etc.).
```

Comment:
Place a comma before "which contains the exhaustive..."
Remove "the" prior to "URIs"